Makefile

```diff
@@ -104,21 +108,21 @@ COVERAGE_PROFILE = $(COVERAGE_DIR)/profile.out
 COVERAGE_XML = $(COVERAGE_DIR)/coverage.xml
 COVERAGE_HTML = $(COVERAGE_DIR)/index.html
 .PHONY: test-coverage test-coverage-tools
-test-coverage-tools: | $(GOCOVMERGE) $(GOCOV) $(GOCOVXML)
+test-coverage-tools: | $(GOCOVMERGE) $(GOCOV) $(GOCOVXML) $(GCOV2LCOV)
```

Comment:
pretty sure we can cleanup some the the test targets like the xml ones, but not related to this PR

Reply:
Do you mean we can drop COVERAGE_XML output? I really have no idea if there are any tools that read it